Makefile

```diff
@@ -6,3 +6,6 @@ fling: fling.c
 
 clean:
 	rm -f fling
+
+check:
```

Comment:
The check target should probably depend on the fling target?

Reply:
Good point. I'll fix.